server/database/data/admin.js

```diff
@@ -0,0 +1,17 @@
+const Admin = require('../models/admin');
+
+module.exports = async () => {
+  const admin = [
+    {
+      id: '5099803df3f4948bd2f98391',
```

Comment:
we do not want the id in here because MongoDB create an id for all instance as this name "_id"